packages/webchat/src/components/messages/MessageGroup.tsx

```diff
@@ -91,47 +54,39 @@ class MessageGroup extends React.Component<Props> {
-              // TODO: use the components package
-              if (payload?.wrapped) {
-                payload = { ...payload, wrapped: renderPayload(payload.wrapped) }
-              } else {
-                payload = renderPayload(payload)
+              let payload = message.payload
+              if (payload.type === 'single-choice') {
+                payload = {
+                  ...payload,
+                  type: 'quick_reply'
+                }
               }
-              */
 
+              // TODO: do we still have inline feedback?
               const showInlineFeedback =
                 isBot && isLastMsg && (payload.wrapped ? payload.wrapped.collectFeedback : payload.collectFeedback)
 
               return (
-                <Message
+                <ReactMessageRenderer
                   key={message.id}
-                  isHighlighted={this.props.highlightedMessages && this.props.highlightedMessages.includes(message.id)}
-                  inlineFeedback={
-                    showInlineFeedback && (
-                      <InlineFeedback
-                        intl={this.props.store!.intl}
-                        messageId={message.id}
-                        onFeedback={this.props.onFeedback}
-                        messageFeedbacks={this.props.store!.messageFeedbacks}
-                      />
-                    )
-                  }
-                  messageId={message.id}
-                  noBubble={!!payload.noBubble}
-                  fromLabel={fromLabel}
-                  isLastOfGroup={i >= this.props.messages.length - 1}
-                  isLastGroup={this.props.isLastGroup}
-                  isBotMessage={!message.authorId}
-                  payload={payload}
-                  sentOn={message.sentOn}
-                  onSendData={this.props.onSendData}
-                  onFileUpload={this.props.onFileUpload}
-                  store={this.props.store}
-                  onAudioEnded={this.onAudioEnded}
-                  shouldPlay={this.state.audioPlayingIndex === i}
+                  content={payload}
+                  config={{
+                    messageId: message.id,
+                    noMessageBubble: !!payload.noBubble,
+                    isLastOfGroup: i >= this.props.messages.length - 1,
+                    isLastGroup: this.props.isLastGroup,
+                    isBotMessage: !message.authorId,
+                    sentOn: message.sentOn,
+                    onSendData: this.props.onSendData,
+                    onFileUpload: this.props.onFileUpload,
+                    store: this.props.store,
+                    onAudioEnded: this.onAudioEnded,
+                    shouldPlay: this.state.audioPlayingIndex === i,
+                    intl: this.props.store!.intl,
+                    escapeHTML: true,
+                    isInEmulator: false,
+                    showTimestamp: this.props.store!.config.showTimestamp
+                  }}
                 />
```

Comment:
Here's the integration in the webchat. There doesn't seem to be an InlineFeedback option was this moved somewhere else? Also some props to have direct matches. For example where does `fromLabel` go?